firefox-ios/Client/Frontend/TrackingProtection/CertificatesViewController.swift

```diff
@@ -252,6 +260,7 @@ extension CertificatesViewController {
         view.backgroundColor = theme.colors.layer5
         certificatesTableView.backgroundColor = .clear
         titleLabel.textColor = theme.colors.textPrimary
+        tableViewHeader.backgroundColor = theme.colors.layer5
```

Comment:
I see you've created a new view to add a background. It seems like a lot of work for this. I don't think this should be necessary... but yes, the theme makes it more complicated.

Can you try a completely different fix instead of the one from this PR?

See if you can just do this one line in `applyTheme()` to solve the problem:

`certificatesTableView.headerView(forSection: 0)?.backgroundColor = theme.colors.layer5`

I'm not sure it will work, but it's worth a shot. :) 

---

A more complicated solution would be to refactor the header view into its own file and UIView class, conforming to protocol `ThemeApplicable`. Then CertificatesViewController can call its `applyTheme` method when creating the headerView in `tableView(headerViewForSection:)`. That way, it will always have the right appearance when created. 

Then in CertificatesViewController's `applyTheme` you could simply call `tableView.reloadData()` to ensure the header gets recreated if the theme changes.